AndroidSDKCore/src/main/java/com/leanplum/internal/Request.java

```diff
@@ -581,6 +581,7 @@ private RequestsWithEncoding getRequestsWithEncodedString() {
   }
 
   private void sendRequests() {
+    CountAggregator.INSTANCE.sendAllCounts();
```

Comment:
Let's discuss throttling this to be less frequent. If we send half as often it means we can increase our sample size 2x.